docs/examples/search-external-access.md

```diff
@@ -0,0 +1,72 @@
+# Enable Alfresco Search Services (`/solr`) External Access
+This example demonstrates how to enable Alfresco Search services for external access.
+
+## Prerequisites
+
+You will need to make sure the [ACS Ingress controller](../helm-deployment-aws_cloud.md#deploying-the-ingress-for-alfresco-content-services) is installed.
+
+## Deployment
+
+By default, Alfresco Search services endpoint (`/solr`) is disabled for external access due to security reasons.  But, this can be enabled should you wish.  You may need to adjust the configuration settings as per [acs-deployment configuration table](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services#configuration).
+
+For full reference to ENVIRONMENT variables used in below snippets, please see [ACS Deployment](../helm-deployment-aws_cloud.md#deploying-alfresco-content-services)
```

Comment:
See ACS Deployment for a full reference to the environment variables used in the snippets below.